bears/generate_package.py

```diff
@@ -40,9 +40,7 @@ def create_file_from_template(template_file, output_file, substitution_dict):
 def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
```

Comment:
these changes are all related to the init.py one, they all are related because changing this also changes entry points and also changes tests :)